lambda/controller/validate-source/index.js

```diff
@@ -29,6 +31,19 @@ exports.handler = function(event, context, callback){
       }
     }
 
+    sourcedb.listTagsOfResource({ ResourceArn: sourceTable.Table.TableArn }, function(err, data) {
+      if(err){
+        return callback(new Error("Tags could not be listed for source table"));
+      } else {
+        for(const tag of data.Tags) {
+          if(tag.Key === GLOBAL_TAG_KEY && tag.Value === GLOBAL_TAG_VALUE) {
+            console.info("Do not replicate source table because it is a global table");
```

Comment:
Might be good print out what table this log message is for. I don't know if it's not a concern because it's all sequential and there's an earlier print of the table name/ARN perhaps?